agent/state.go

```diff
@@ -10,6 +10,7 @@ import (
 	mglog "github.com/absmach/magistrala/logger"
 )
 
+//go:generate stringer -type=state
```

Comment:
What's this?

Reply:
states are defined as `int` we use go generate stringer to generate string of the constants for submission of events https://pkg.go.dev/golang.org/x/tools/cmd/stringer